typedoc.json

```diff
@@ -0,0 +1,69 @@
+{
+	"$schema": "https://typedoc.org/schema.json",
+	"categoryOrder": [
+		"Node Types",
+		"Nodes - Type Guards",
+		"Nodes - Flag Utilities",
+		"Nodes - Other Utilities",
+		"Modifier Utilities",
+		"Scope Utilities",
+		"Syntax Utilities",
+		"Compiler Options",
+		"*",
+		"Other"
+	],
+	"customCss": "./typedoc.css",
```

Comment:
[Suggestion] It might make sense to move the file into `src/`? The root path is normally for config files. But I don't feel strongly here, certainly not enough to block.

Reply:
I don't like the idea of putting it in `src/` as it's not a source file for our library itself. Maybe putting it in `docs/` instead.